packages/blocks/src/api/index.js

```diff
@@ -67,3 +67,11 @@ export {
 } from './templates';
 export { default as children } from './children';
 export { default as node } from './node';
+export {
+	COLOR_SUPPORT_KEY,
+	FONT_SIZE_SUPPORT_KEY,
+	LINE_HEIGHT_SUPPORT_KEY,
+	LINK_COLOR,
+	PADDING_SUPPORT_KEY,
+	STYLE_PROPERTY,
+} from './constants';
```

Comment:
I'm concerned about this personally, it makes it an API and I don't really know whether these keys will remain as is, be combined... Why do we need this to be an API? (specially the `_key` ones)

Reply:
mmm, my rationale was that the support keys are actually part of the API. This is why: if we had specific accessors (`hasLineHeightSupport`, `hasFontSizeSupport`, etc), consumers wouldn't need anything else to use them; however, by having general accessors (such as `hasBlockSupport( feature )`) we also need to provide the consumer with a list of the available keys they can use. So far, they inspect the block.json for this. I'd think providing constants for them would make it easier for us to modify their names later on if we need to. Said this, I realize we don't expose any other feature, so I won't hold this opinion too strongly and would be fine to move these back to block-editor, if that's preferred.

For the style keys, I agree with Jorge that we need a single source of truth, as they're both used in block-editor and edit-site. Exporting them here makes more sense from my POV.

Reply:
in which situation we need to access these keys outside the hooks themselves which are part of block-editor package?

Reply:
I reverted the support keys to block-editor. I still think that we should offer a feature list as I mentioned, but I realized that it doesn't seem a good idea to start with the experimental keys.